web/meetup-facilitator/src/features/GoogleAuth.jsx

```diff
@@ -1,23 +1,25 @@
 import { GoogleLogin } from '@react-oauth/google'
-import { useState} from 'react';
+import { useState  } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 function GoogleAuth() {
 
-    const [email, setEmail] = useState('');
-    const [name, setName] = useState('');
-    const [pictureURL, setPictureURL] = useState('');
-    const [isAuthorized, setIsAuthorized] = useState(false);
-
+    const navigate = useNavigate();
+    
 
     const responseMessage = (response) => {
         console.log(response);
         const decodedJWT = JSON.parse(atob(response.credential.split('.')[1]))
         console.log(decodedJWT);
         if(Object.keys(decodedJWT).length === 15) {
-            setEmail(decodedJWT.email);
-            setName(decodedJWT.given_name + ' ' + decodedJWT.family_name);
-            setPictureURL(decodedJWT.picture);
-            setIsAuthorized(true);
+
+            sessionStorage.setItem('name', decodedJWT.given_name + ' ' + decodedJWT.family_name);
```

Comment:
Let's use `localStorage` instead of `sessionStorage` to store the user info, this will allow the user to stay signed in even after restarting the browser or their computer.